packages/list/package.json

```diff
@@ -31,13 +31,13 @@
     "url": "https://github.com/prateekbh/preact-material-components/issues"
   },
   "dependencies": {
-    "@material/list": "0.39.3",
+    "@material/list": "1.1.1",
```

Comment:
Do all components have the same version? I saw some with `1.1.0`

Reply:
There might have been a release recently we'll upgrade all in the end once

Reply:
That's fine